portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,53 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private List<String> commentParts;
```

Comment:
Since commentParts is only used in doPost, it is better to put it as a function variable instead of class member, plus the value is being overwritten each time.

Reply:
I think it's used in doGet `String json = convertToJsonUsingGson(commentParts);`

Reply:
Yup, I believe it is used in both functions